src/logics/uniswap-v3/configs.ts

```diff
@@ -26,20 +26,29 @@ export const configs: univ3.Config[] = [
       [mainnetTokens.renBTC.address]: [mainnetTokens.WBTC],
     },
   },
+  {
+    chainId: common.ChainId.optimism,
+    factoryAddress: '0x1F98431c8aD98523631AE4a59f267346ea31F984',
+    quoter: { address: '0x61fFE014bA17989E743c5F6cB21bF9697530B21e', isV2: true },
+    swapRouterAddress: '0xE592427A0AEce92De3Edee1F18E0157C05861564',
+    feeAmounts: [FeeAmount.LOWEST, FeeAmount.LOW, FeeAmount.MEDIUM, FeeAmount.HIGH],
+    bases: [optimismTokens.OP, optimismTokens.DAI, optimismTokens['USDC.e'], optimismTokens.USDT, optimismTokens.WBTC],
```

Comment:
Wondering that if we should change the naming to make `toTokenMap` works for `USDC.e` instead of using a dictionary key.

Reply:
I asked @chouandy a similar Q last time and he said this is another legit way to do it in JS/TS. I think it's easier not to add another function to do the mapping. What does @chouandy think about this?